kong/llm/schemas/init.lua

```diff
@@ -68,6 +81,20 @@ local auth_schema = {
                       "environment variable `GCP_SERVICE_ACCOUNT`.",
         required = false,
         referenceable = true }},
+    { aws_access_key_id = {
+        type = "string",
+        description = "Set this if you are using an AWS provider (Bedrock, SageMaker) and you are authenticating " ..
+                      "using static IAM User credentials.",
```

Comment:
let's mention this overwrites the gloabl AWS_ACCESS_KEY_ID env var (same for the access_key field below)